docs/examples/rust/exchange/get-balance.rs

```diff
@@ -0,0 +1,14 @@
+use std::str::FromStr;
+use iota_sdk::types::base_types::IotaAddress;
+use iota_sdk::IotaClientBuilder;
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+   let iota = IotaClientBuilder::default().build(
+      "https://api.devnet.iota.cafe:443",
+   ).await.unwrap();
```

Comment:
Btw why don't use testnet? Also fmt fails
```suggestion
   let iota = IotaClientBuilder::default().build_testnet().await?;
```

Reply:
No clue dbf 🤔 
Guess back then we only had devote 😅